### PR TITLE
Request to review the code of new MF Test cases

### PR DESCRIPTION
**Pull request by sayantanigoswami1996**

### CODE REVIEW

File: AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+${e_portfolio_mf_noOfFundsLabel} =  No. of Funds
+${e_portfolio_mf_invest_searchHint} =  Type fund to invest in
+${e_portfolio_mf_investSearchInput} =  Tata Small Cap Growth Direct Plan
+
+
+# Application Locator 'KU_<locatorName>'
+# Portfolio Tab Navigation Page
+${KU_W_portfolio_mf_value} =  xpath=//div[@class='b-portfolio-tab__value b-portfolio-tab__value--selected']
+${KU_W_portfolio_mf_totalValLabel} =  xpath=//p[@class='b-cumulative-fund-row__total-label']
+${KU_W_portfolio_mf_totalVal} =  xpath=//*[@class='b-cumulative-fund-row__total-value-rs-data']
```
Comment (shantanu-kuvera) [inline AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot:16]: Avoid using * as it searches the entire dom for the element. Use tags instead. 
Remove * for element search from the entire file.

File: AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+${e_portfolio_mf_dayReturnLabel} =  1 Day Return
+${e_portfolio_mf_currReturnLabel} =  Current Return
+${e_portfolio_mf_noOfFundsLabel} =  No. of Funds
+${e_portfolio_mf_invest_searchHint} =  Type fund to invest in
+${e_portfolio_mf_investSearchInput} =  Tata Small Cap Growth Direct Plan
+
+
+# Application Locator 'KU_<locatorName>'
+# Portfolio Tab Navigation Page
+${KU_W_portfolio_mf_value} =  xpath=//div[@class='b-portfolio-tab__value b-portfolio-tab__value--selected']
```
Comment (shantanu-kuvera) [inline AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot:14]: Avoid using class as the attribute when there is a compounded string. Please check whether some other attribute can be incorporated here.

**Review by shantanu-kuvera — CHANGES_REQUESTED** [review]

File: AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot (hunk 1 above)
Comment (sayantanigoswami1996) [reply AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot:16]: This has been fixed . The * has been replaced with respective tags.

File: AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot (hunk 2 above)
Comment (sayantanigoswami1996) [reply AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot:14]: As discussed , this has been understood from the two solutions which has been given by you . Henceforth , will be taken care of when freshly coded. Old codebase can be changed later.

File: AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot (hunk 1 above)
Comment (PrathijaEtagi) [reply AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot:16]: Fixed

File: AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot (hunk 2 above)
Comment (PrathijaEtagi) [reply AppLocators/DesktopWeb/PostLoginFlowsLocators/MutualFunds/PortfolioMFLocators.robot:14]: Looks fine

**Review by PrathijaEtagi — APPROVED** [review]
Approved

**Comment by sayantanigoswami1996** [comment]
> Approved, Take care of confilts.

Yes.

**Pull request closed without merging**